frontend/src/components/ChatBox/ChatBoxMessage/ChatBoxMessage.tsx

```diff
@@ -0,0 +1,34 @@
+import { CHAT_MESSAGE_TYPE, ChatMessage } from '@src/models/chat';
+
+import Avatar from './Avatar';
+import MessageBubble from './MessageBubble';
+
+import './ChatBoxMessage.css';
+
+function ChatBoxMessage({ message }: { message: ChatMessage }) {
+	const avatar =
+		message.type === CHAT_MESSAGE_TYPE.USER ||
+		message.type === CHAT_MESSAGE_TYPE.USER_TRANSFORMED
+			? 'user'
+			: message.type === CHAT_MESSAGE_TYPE.BOT
+			? 'bot'
+			: message.type === CHAT_MESSAGE_TYPE.BOT_BLOCKED ||
+			  message.type === CHAT_MESSAGE_TYPE.ERROR_MSG
+			? 'botError'
+			: 'none';
+
+	const owner = avatar === 'botError' ? 'bot' : avatar;
+
+	const position =
+		owner === 'user' ? 'right' : owner === 'bot' ? 'left' : 'centre';
```

Comment:
Very minor, but I personally found this a little tricky to read. Might as well not use `owner` imo and just put it right into the `position` conditional.
```suggestion
	const position =
			avatar === 'user' 
			? 'right' 
			: avatar === 'bot' || avatar === 'botError'
			? 'left' 
			: 'centre';
```

Reply:
Ah, a relic from a previous implementation :) good spot